examples/prediction_upload/video_predictions.ipynb

```diff
@@ -64,18 +64,7 @@
         "!pip install -q 'labelbox[data]'"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    model_run.upsert_data_rows(global_keys=global_key)
</pre><p>Might be better practice to pass in a list of global_keys here, since the function accepts a list</p><pre class="ql-syntax" spellcheck="false">model_run.upsert_data_rows(global_keys=[global_key])
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1006/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>